versions/baseline.json

```
@@ -1770,7 +1770,7 @@
     },
     "curl": {
       "baseline": "7.84.0",
-      "port-version": 0
+      "port-version": 2
```

```suggestion
      "port-version": 1
```
Please bump the port version. See [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md#update-the-version-field-in-the-control-file-of-any-modified-ports).